app/src/test/java/org/simple/clinic/login/pin/LoginPinScreenLogicTest.kt

```diff
@@ -128,7 +128,8 @@ class LoginPinScreenLogicTest {
         loggedInStatus = User.LoggedInStatus.OTP_REQUESTED,
         registrationFacilityUuid = registrationFacilityUuid,
         currentFacilityUuid = registrationFacilityUuid,
-        teleconsultPhoneNumber = teleconsultPhoneNumber
+        teleconsultPhoneNumber = teleconsultPhoneNumber,
+        capabilities = null
```

Comment:
You can do it in next PR, can we also pass capabilities here? Look at how we passed `teleconsultPhoneNumber`